variantrule/James Haeck; Traveling with Style Skill Challenges.json

```diff
@@ -0,0 +1,193 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "TWS:SC",
+				"abbreviation": "TWS:SC",
+				"full": "Traveling with Style: Skill Challenges",
+				"authors": [
+					"James Haeck"
+				],
+				"convertedBy": [
+					"Lyra [he/him]"
+				],
+				"version": "2018-07-30",
+				"color": "47D18C",
+				"url": "https://www.dndbeyond.com/posts/285-traveling-with-style-skill-challenges",
+				"dateReleased": "2018-07-30"
+			}
+		],
+		"status": "ready",
+		"dateAdded": 1706088493,
+		"dateLastModified": 1706088493
+	},
+	"variantrule": [
+		{
+			"name": "Traveling with Style: Skill Challenges",
+			"source": "TWS:SC",
+			"ruleType": "O",
+			"entries": [
+				{
+					"type": "section",
+					"entries": [
+						"Yesterday, my players' characters needed to travel for a week from a desolate dungeon to a thriving city. I knew there was only one way to make this journey exciting for everyone: a skill challenge. This article is a follow-up to {@link Exploring the Wilderness: Navigation and Player Agency|https://www.dndbeyond.com/posts/269-exploring-the-wilderness-navigation-and-player} and {@link Exploring the Wilderness: Creating Dynamic Encounter Areas|https://www.dndbeyond.com/posts/272-exploring-the-wilderness-creating-dynamic}. In the former article, I explored many different ways of making travel interesting, and posed a basic outline for using skill challenges to engage your players during travel sequences. This week, we're taking a more detailed look at how to make this much-misunderstood system work.",
+						"I love making maps, especially overland maps on the scale of city-states, nations, and continents. I spent hours upon hours memorizing the tiniest details of Brian Jacques' {@i Redwall} maps, and the map of Bilbo's journey in {@i The Hobbit}, and then was astounded to see it expanded even further in {@i The Lord of the Rings}. But in roleplaying games, maps can't just look pretty or inspire the reader's imagination, as they might do in a novel. In RPGs, the players have to use the map to guide themselves on adventures, and that map must be filled with beings and locations. Otherwise, why use a map? If the only thing that matters are the points and destinations, you would be better off saving time and effort and just using a flowchart.",
+						{
+							"type": "image",
+							"href": {
+								"type": "external",
+								"url": "https://www.dndbeyond.com/attachments/thumbnails/4/181/200/523/poa01-05.png"
```

Comment:
😟

Reply:
I guess that one's possible as an internal url, also you want them changed? It just felt trivial to grab the links from a public article